ci/pipeline.yml

```diff
@@ -10,21 +10,27 @@ variables:
 
 build_job:
   stage: build
-  extends: .container-builder
+  extends: .container-builder-cscs-gh200
   variables:
     DOCKERFILE: ci/docker/Dockerfile.build
 
 test_job:
   stage: test
-  extends: .container-runner-daint-gpu
+  extends: .container-runner-daint-gh200
   image: $PERSIST_IMAGE_NAME
   script:
+    - export LD_LIBRARY_PATH=/usr/lib64:$LD_LIBRARY_PATH
+    - export LD_PRELOAD=/usr/lib64/libmpi_gtl_cuda.so
     - ctest --test-dir /quda/build/ --output-on-failure
   variables:
     CRAY_CUDA_MPS: 0
-    SLURM_JOB_NUM_NODES: 2
+    NVIDIA_VISIBLE_DEVICES: all
+    SLURM_JOB_NUM_NODES: 1
+    SLURM_NTASKS: 2
```

Comment:
since we have 4 gpus per node, we could change the test setup to 4 gpus without changing resource usage...